cli/wallet/wallet.go

```diff
@@ -46,13 +49,41 @@ func NewCommand() cli.Command {
 						Name:  "path, p",
 						Usage: "Target location of the wallet file.",
 					},
+					cli.StringFlag{
+						Name:  "passphrase, pass",
+						Usage: "Passphrase to unlock an account in the wallet.",
+					},
 				},
 			},
 		},
 	}
 }
 
 func openWallet(ctx *cli.Context) error {
+	path := ctx.String("path")
+	if len(path) == 0 {
+		return cli.NewExitError(errPathMandatory, 1)
+	}
+
+	pass := ctx.String("pass")
```

Comment:
It's generally best to avoid people entering their password as part of a command so it doesn't end up in their bash history, this would be better read from `os.Stdin` rather than a cli argument.